src/control/lib/control/pool.go

```diff
@@ -717,3 +732,47 @@ func PoolReintegrate(ctx context.Context, rpcClient UnaryInvoker, req *PoolReint
 
 	return nil
 }
+
+// VerifyPolicyString will parse the incoming policy string and return
+// a flag whether a combination of policy type and its parameters is valid
+// Returns an error if policy string is wrong, policy can't be found or policy parameters are wrong
+func VerifyPolicyString(stringPolicy string) (err error) {
+
+	sp := strings.Split(strings.ToLower(stringPolicy), ",")
+
+	var policystr string = ""
+	var policyparams string = ""
+
+	for _, paramstr := range sp {
+		param_split := strings.Split(paramstr, "=")
```

Comment:
I think this could be improved by supplying PolicyType and PolicyParams separately in the pool create command. Again, just a suggestion.